Boa.Constrictor.RestSharp.UnitTests/Serialization/FullRestDataTest.cs

```diff
@@ -65,7 +65,8 @@ public void SetUp()
                 new RestClientOptions
                 {
                     ConfigureMessageHandler = _ => mockHttp,
-                    BaseUrl = ClientUri
+                    BaseUrl = ClientUri,
+                    CookieContainer = new CookieContainer()
```

Comment:
Does this `CookieContainer` come from `RichardSzalay.MockHttp`?

Reply:
`CookieContainer` is a RestSharp object. It existed before the recent changes. Just moved from `restClient.CookieContainer` to `RestClient.Options.CookieContainer`